front-end/src/app/reports/f3x/create-workflow/create-f3x-step3.component.ts

```diff
@@ -31,6 +31,12 @@ export class CreateF3xStep3Component extends TableListBaseComponent<Transaction>
   protected getEmptyItem(): Transaction {
     return {} as Transaction;
   }
+
+  override getGetParams(): { [param: string]: string | number | boolean | ReadonlyArray<string | number | boolean> } {
+    const reportId = this.activatedRoute.snapshot.params['reportId'];
+    const params = { report_id: reportId };
```

Comment:
What happens if the url does not have a value for :reportId? Does an error get thrown?

Reply:
The routing requires a value for :reportId.  If no value is provided, it redirects you to the Manage Reports page.  If the value is not a number ("x" was used in testing), then it evaluates to "api/v1/f3x-summaries/NaN/" and you get a 404 error.